### ISSUE TITLE
multicloud issue

### ISSUE THREAD

**Issue by SocialHulkHogan** [WheatonCS/Lexos#343]
Going to the url http://localhost:5000/multicloud on firefox before going to upload page causes a crash. to recreate this, go there on firefox. It's gunna crash.

**Comment by SocialHulkHogan** [comment]
I think I was just being really dumb

**Comment by scottkleinman** [comment]
Not really. I've done that before when trying to do topic clouds. Since Multicloud has a separate upload tool for MALLET files, it is tempting to go there directly before starting a session through the Upload page. A user is less likely to do this for other pages, but we should still catch this and re-direct the user to Upload if there is no session. I'll open this as a separate issue.
